src/pages/patientView/copyNumberAlterations/column/AnnotationColumnFormatter.tsx

```diff
@@ -96,7 +96,7 @@ export default class AnnotationColumnFormatter
         let geneVariants:{[name: string]: ICivicVariantData} = getCivicCNAVariants(copyNumberData, geneSymbol, civicVariants);
         let geneEntry: ICivicGeneData = civicGenes[geneSymbol];
         //Only return data for genes with variants or it has a description provided by the Civic API
-        if (!_.isEmpty(geneVariants) || geneEntry && geneEntry.description !== "") {
+        if (!_.isEmpty(geneVariants) && geneEntry || geneEntry && geneEntry.description !== "") {
```

Comment:
i think we should put in some parenthesis to make this clearer

Reply:
Thanks, updated!